server/routes/key-worker-profile/controller.ts

```diff
@@ -14,18 +13,13 @@ export class KeyWorkerProfileController {
 
     const keyworkerName = `${keyworkerData.keyworker.firstName} ${keyworkerData.keyworker.lastName}`
     const keyworkerStatus = keyworkerData.status.description
-    const keyworkerDetails = this.formatDetails(keyworkerData)
-    const keyworkerStats = this.formatStats(keyworkerData.stats.current, keyworkerData.stats.previous)
     const allocationRecords = this.mapAllocationsToRecords(keyworkerData.allocations)
```

Comment:
mapAllocationsToRecords doesn't really do much other than rename some properties and format them for the view. The view still as to create the rows. Perhaps consider doing it all in the view?